src/base/bittorrent/torrenthandle.h

```diff
@@ -315,6 +316,14 @@ namespace BitTorrent
         void setName(const QString &name);
         void setSequentialDownload(bool b);
         void toggleSequentialDownload();
+        // disables deadlines if completionTime < 0
+        void scheduleDownloading(std::chrono::milliseconds totalDeadline,
+                                 std::chrono::milliseconds delay = std::chrono::milliseconds(0),
```

Comment:
As far as I remember, GCC 5.3 disagreed to compile that. Would be nice to use `0ms`, but that requires
```C++
using namespace std::literals::chrono_literals;
```